bk-api/bk_api.py

```diff
@@ -724,10 +677,9 @@ def deploy_wes(node_id, this_debug, force=False):
     if not beehive_id:
         raise Exception(f"beehive_id is missing")
 
-    logger.debug(f"(deploy_wes) using beehive {beehive_id}")
+    logger.debug("(deploy_wes) using beehive %s", beehive_id)
 
-
-    # first check if kubectl work on the node
+    logger.debug("(deploy_wes) checking if node %s is running k8s", node_id)
```

Comment:
its not `k8s` is it?  `k3s`?  

too picky? :p

Reply:
haha.. how about just kubernetes :)